### ISSUE TITLE
Could you please check if this is wrong or not?

### ISSUE THREAD

**Issue by leeky940926** [django-polymorphic/django-polymorphic#511]
Hi, I'm Lee-Kiyong who works as a django developer in South Korea

First of all, I'm sorry that I have to write something trivial.

I found it while reading the documentation to study polymorphic.

In this [link](https://github.com/django-polymorphic/django-polymorphic/blob/5111fb070e4e0b0153e816d163871f2039ca2ae2/docs/advanced.rst), Is this what you meant?

```real_objects=ModelA.objects.get_real_instances(base_objects_queryset_or_object_list)```

<br>

<img width="967" alt="image" src="https://user-images.githubusercontent.com/88086271/155873642-320527a7-9f0e-4533-9aa3-12da3f3030e4.png">

**Comment by AdamDonna** [comment]
Yeah I see the same rendering issue. Thanks for bringing that to our attention 👍

**Comment by bckohan** [comment]
This was fixed at some point.
